File: easy/go/negatives.go
package mostFrequent

func countNegatives(grid [][]int) int {	
	var negatives int32	
	for _,arr := range grid{ 
		for _,val := range arr{ 
			if val <0{ 
				negatives +=1
			}
		}
	}
	return int(negatives)
}

File: easy/rust/negatives.rs
impl Solution {
    pub fn count_negatives(grid: Vec<Vec<i32>>) -> i32 {
        
        let mut negatives : i32 = 0;
        for arr in grid.iter(){ 
            for val in arr.iter(){ 
                if *val  <0 { 
                    negatives +=1 ;
                }
            }
        }
        negatives
    }
    
}
